File: dev/ProyectoCadima/Entidades/Seguridad/Empresa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entidades.Seguridad
{
    public class Empresa
    {
        public int EmpresaId { get; set; }
        public int TipoIdc { get; set; }
        public string Nit { get; set; }
        public string Nombre { get; set; }
        public string Gerente { get; set; }
        public string Telefono { get; set; }
        public bool Estado { get; set; }
        public Empresa() { }
    }
}

File: dev/ProyectoCadima/android/app/src/main/java/com/nur/compraonline/model/security/Empresa.java
package com.nur.compraonline.model.security;

import com.nur.compraonline.model.Entity;

/**
 * Created by dpedrazas on 12/09/2017.
 */

public class Empresa extends Entity {

    private Integer EmpresaId ;
    private Integer TipoIdc ;
    private String Nit ;
    private String Nombre ;
    private String Gerente ;
    private String Telefono ;
    private Boolean Estado ;

    public Empresa() {
    }

    public Integer getEmpresaId() {
        return EmpresaId;
    }

    public void setEmpresaId(Integer empresaId) {
        EmpresaId = empresaId;
    }

    public Integer getTipoIdc() {
        return TipoIdc;
    }

    public void setTipoIdc(Integer tipoIdc) {
        TipoIdc = tipoIdc;
    }

    public String getNit() {
        return Nit;
    }

    public void setNit(String nit) {
        Nit = nit;
    }

    public String getNombre() {
        return Nombre;
    }

    public void setNombre(String nombre) {
        Nombre = nombre;
    }

    public String getGerente() {
        return Gerente;
    }

    public void setGerente(String gerente) {
        Gerente = gerente;
    }

    public String getTelefono() {
        return Telefono;
    }

    public void setTelefono(String telefono) {
        Telefono = telefono;
    }

    public Boolean getEstado() {
        return Estado;
    }

    public void setEstado(Boolean estado) {
        Estado = estado;
    }
}
